Add R16 release notes (#210)
pxblue/doc-it: src/docs/release-notes/R16/Summary.mdx

> Added new items to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries, added search functionality to our documentation site, introduced reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native, and publicized our stickersheet in Figma.

Reviewer: ```suggestion
Added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries, added search functionality to our documentation site, introduced reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native, and published our stickersheet to the Figma community.
```
